tests/system/Config/fixtures/.env

```diff
@@ -32,3 +32,10 @@ SimpleConfig.crew.captain = Malcolm
 SimpleConfig.crew.pilot = Wash
 SimpleConfig.crew.comms = true
 SimpleConfig.crew.doctor = false
+
+SimpleConfig.float = '0.0'
+SimpleConfig.int = '999'
```

Comment:
Per your https://github.com/codeigniter4/CodeIgniter4/pull/5704#discussion_r808983149, this should result in strings since encapsed in quotes. However, in lines 64-65 of the tests, those are tested as float and int respectively.

Isn't it contradictory? The reason why my PR was stalled is because of trying to find a solution to that distinction (i.e., string 12 and int 12) and this PR is not addressing that.

Reply:
Yes, I've changed my mind.

After all, the correct type is the property type in the Config class.

But I don't recommend to use `'` for int/float value, I change the test fixture.